apps/camera/js/controllers/preview-gallery.js

```diff
@@ -137,7 +137,7 @@ PreviewGalleryController.prototype.onOptionsClick = function() {
 };
 
 PreviewGalleryController.prototype.shareCurrentItem = function() {
-  if (this.app.inSecureMode) {
+  if (this.app.inSecureMode || this.resizingImage) {
```

Comment:
I tried using pointer-events: none instead to disable the preview gallery menu (via the busy/ready signals) but an extra resize event could still make it through sometimes before the class gets applied to the element.